core/src/test/java/com/gentics/mesh/test/context/MeshTestContext.java

```diff
@@ -500,6 +500,7 @@ public MeshOptions init(MeshTestSetting settings) throws Exception {
 			break;
 		case TRACKING:
 			System.setProperty(TrackingSearchProvider.TEST_PROPERTY_KEY, "true");
+		case BROKEN: // fall-through
```

Comment:
Change type to "UNREACHABLE". Add searchOptions.setUrl("http://localhost:1") - Remove the other new search provider code.